keras/layers/convolutional_recurrent.py

```diff
@@ -494,7 +496,7 @@ def input_conv(self, x, w, b=None, padding='valid'):
                                   data_format=self.data_format)
         return conv_out
 
-    def reccurent_conv(self, x, w):
+    def recurrent_conv(self, x, w):
```

Comment:
`reccurent_conv` was a typo, but it is now part of the API. Keep it as an alias for `recurrent_conv` (with deprecation warning?)